gateway/api/v1/views/jobs_new/runtime_jobs.py

```diff
@@ -0,0 +1,111 @@
+"""
+API endpoint to handle runtime jobs.
+"""
+
+# pylint: disable=abstract-method
+
+from typing import Any
+from uuid import UUID
+
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import permissions, serializers, status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from api import serializers as api_serializers
+from api.use_cases.jobs.associate_runtime_jobs import (
+    AssociateRuntimeJobsUseCase,
+)
+from api.use_cases.jobs.get_runtime_jobs import GetRuntimeJobsUseCase
+from api.v1.endpoint_decorator import endpoint
+from api.v1.endpoint_handle_exceptions import endpoint_handle_exceptions
+from api.v1.views.swagger_utils import standard_error_responses
+
+
+class InputSerializer(serializers.Serializer):
+    """
+    Validates the request body for associating runtime jobs to a serverless job.
+    """
+
+    runtime_job = serializers.CharField(required=True)
+    runtime_session = serializers.CharField(
+        required=False, allow_null=True, allow_blank=True
+    )
+
+    class Meta:
+        """Meta class to define input serializer name"""
+
+        ref_name = "RuntimeJobsInputSerializer"
+
+
+class RuntimeJobSerializer(api_serializers.RuntimeJobSerializer):
+    """
+    Runtime job serializer fileds.
+    """
+
+    class Meta(api_serializers.RuntimeJobSerializer.Meta):
+        fields = ["runtime_job", "runtime_session"]
+
+
+def serialize_output(out_runtime_jobs: Any) -> dict[str, Any]:
+    """
+    Build the response payload for runtime_jobs.
+
+    Args:
+        job: Updated job instance.
+
+    Returns:
+        A dictionary containing the serialized runtime jobs under the 'runtime_jobs' key.
+    """
+    return {"runtime_jobs": RuntimeJobSerializer(out_runtime_jobs, many=True).data}
+
+
+@swagger_auto_schema(
+    method="get",
+    operation_description="Retrieve runtime jobs associated with a job.",
+    responses={
+        status.HTTP_200_OK: RuntimeJobSerializer,
+        **standard_error_responses(
+            not_found_example="Job [XXXX] not found",
+            unauthorized_example="Authentication credentials were not provided or are invalid.",
+        ),
+    },
+)
+@swagger_auto_schema(
+    method="post",
+    operation_description="Associate a runtime job with a job.",
+    request_body=InputSerializer,
+    responses={
+        status.HTTP_200_OK: serializers.Serializer(),
+        **standard_error_responses(
+            bad_request_example="Got empty `runtime_job` field. Please, specify `runtime_job`.",
+            not_found_example="Job [XXXX] not found",
+            unauthorized_example="Authentication credentials were not provided or are invalid.",
+        ),
+    },
+)
+@endpoint("jobs/<uuid:job_id>/runtime_jobs", name="jobs-runtime-jobs")
```

Comment:
@korgan00 , @ElePT how the end-point is identified with this I would like to divide this end-point in two, one for GET and another one for POST (two separate files).